src/Resources/js/views/mealIndex.ts

```diff
@@ -0,0 +1,213 @@
+import {ParticipationPreToggleHandler} from "../modules/participation-pre-toggle-handler";
+import {ParticipationToggleHandler} from "../modules/participation-toggle-handler";
+import {ParticipationCountUpdateHandler} from "../modules/participation-count-update-handler";
+import {CombinedMealDialog, Dish, DishVariation} from "../modules/combined-meal-dialog";
+import {ParticipationRequest, ParticipationRequestHandler} from "../modules/participation-request-handler";
+
+export default class MealIndexView {
+    participationPreToggleHandler: ParticipationPreToggleHandler;
+    readonly participationCheckboxSelector: string = '.meals-list .meal .participation-checkbox';
+
+    constructor() {
+        this.updateSlots();
+        setInterval(this.updateSlots, 3000);
+
+        this.initEvents();
+
+        if ($(this.participationCheckboxSelector).length > 0) {
+            console.log($(this.participationCheckboxSelector).length);
+            let $participationCheckboxes = $(this.participationCheckboxSelector);
+            let participationToggleHandler = new ParticipationToggleHandler($participationCheckboxes);
+            let participationCountUpdateHandler = new ParticipationCountUpdateHandler($participationCheckboxes);
+            this.participationPreToggleHandler = new ParticipationPreToggleHandler(participationToggleHandler);
+        } else {
+            console.log($(this.participationCheckboxSelector).length);
```

Comment:
Error message instead?

Reply:
This was just for debugging. It has been removed now.